src/datachain/lib/dc/records.py

```diff
@@ -94,7 +94,11 @@ def read_records(
         {c.name: c.type for c in columns if isinstance(c.type, SQLType)},
     )
     records = (adjust_outputs(warehouse, record, col_types) for record in to_insert)
-    for chunk in batched(records, INSERT_BATCH_SIZE):
```

Comment:
NOt related as well. I guess, INSERT_BATCH_SIZE was imported from there just to have some limit. But these params are not really have the same meaning.

I'd introduce a separate constant in this file instead of importing INSERT_BATCH_SIZE.

Reply:
I am not sure, given that it also inserts the rows into the warehouse, it could be good to have chunk as well. But reverted it back for now.